internal/transport/worker/worker.go

```diff
@@ -0,0 +1,46 @@
+package worker
+
+import (
+	"fmt"
+	"log"
+	"sync"
+	"time"
+
+	"github.com/shipherman/gophermart/internal/clients"
+	"github.com/shipherman/gophermart/internal/db"
+)
+
+type Worker struct {
+	CloseCh chan bool
+	ErrCh   chan error
+	Client  db.DBClientInt
+}
+
+func New(dbc db.DBClientInt) *Worker {
+	close := make(chan bool, 1)
+	err := make(chan error)
+	return &Worker{CloseCh: close, ErrCh: err, Client: dbc}
+}
+
+func (w *Worker) Run(wg *sync.WaitGroup) {
+	for {
+		time.Sleep(time.Second * 10)
```

Comment:
```suggestion
		time.Sleep(config.ArrcuralWorkerInterval)
```
нужно стараться выносить в конфиг все что может повлиять на работу приложения